src/java/htsjdk/samtools/SamReader.java

```diff
@@ -57,6 +57,7 @@ public String toString() {
             }
         }
 
+        public static Type SRA_TYPE = new TypeImpl("SRA", "sra", null);
```

Comment:
We don't have SRA index (there are couple of indexes inside SRA archive itself). We only emulate it in SRAIndex